imap_processing/tests/conftest.py

```diff
@@ -523,18 +524,25 @@ def make_data(start_met: float, end_met: Optional[float] = None) -> pd.DataFrame
             end_met = start_met + 86400
 
         # Create spin start second data of 15 seconds increment
-        spin_start_sec = np.arange(np.floor(start_met), end_met + 1, 15)
+        spin_start_met = np.arange(start_met, end_met + 1, 15)
+        spin_start_sec = np.floor(spin_start_met).astype(int)
         spin_start_subsec = int((start_met - spin_start_sec[0]) * 1000)
 
+        # Calculate UTC times without spice (accepting ~5 second inaccuracy)
+        spin_start_dt64 = TTJ2000_EPOCH + (spin_start_met * 1e9).astype(
```

Comment:
Can we load up the `utc` value here as a dt64? Then I think we would get the correct accuracy because they are using SPICE already.

edit: I see, we are in the testing directory, so these aren't the files we were given they are interal to us. Ignore this comment, just leaving it to note my lack of reading context properly!